src/components/atoms/form/ListProperty.tsx

```diff
@@ -91,9 +91,9 @@ const ListProperty = ({
                     ref={provided.innerRef}
                     className="p-2 pr-0"
                   >
-                    {values?.map((cmd: any, index: number) => (
+                    {Object.keys(values[0])?.map((cmd: any, index: number) => (
```

Comment:
Does it make more sense to keep the conditional values?.map here?

Reply:
Good call. Surprised the linter didn't pick that up. Fixed